ci/workflows/main-cron.yml

```diff
@@ -21,15 +21,15 @@ auto-retry: &auto-retry
 
 steps:
   - label: "build"
-    command: "ci/scripts/build.sh -t ci-dev -p ci-dev"
+    command: "ci/scripts/build.sh -t ci-release -p ci-release"
```

Comment:
Do you know why we use ci-dev in the first place? Just want to make sure changing to ci-release doesn't cause other implicit issue.

Reply:
ci-release will compile the release build, it is the only difference. Because the ingestion phase will be time-consuming if we use debug build.